kahuna/public/js/errors/global.html

```diff
@@ -46,4 +46,9 @@
         A server error has occurred; the Grid may not be fully functional at this time.
         If the problem persists please <a href="{{ctrl.supportEmailLink}}" class="coloured-link">email the Grid team</a>.
     </div>
+
+    <div class="global-error error" ng-message="quarantineFailed">
+        Image is infected with a virus!
```

Comment:
Should the messaging be driven by what we [store in the database](https://github.com/guardian/grid/blob/f50d816f1d236f0145c1ee84f92a30c721bcab8b/image-loader/app/model/UploadStatus.scala#L20)?

Not sure this should be a global error as it's localised to a specific operation - can we position it locally to the image upload? [This](https://github.com/guardian/grid/blob/f50d816f1d236f0145c1ee84f92a30c721bcab8b/kahuna/public/js/upload/jobs/upload-jobs.html#L23-L32) should allow for messages to be shown next to the image(s) affected.